### DISCUSSION TITLE
Data providers

### DISCUSSION THREAD

**Discussion by mraiffe** [kahlan/kahlan#414] [Q&A] [answered]
Hello,
I just test kahlan for few hours but I did not find how to create data providers in specs. Is it possible ?

Thanks,
Mathieu

**Comment by samsonasik** [comment, marked as answer]
You can use foreach instead, answered at issue https://github.com/kahlan/kahlan/issues/146

**Reply by mraiffe** [reply]
Thanks a lot for the answer.
